app/src/main/java/app/revanced/manager/ui/screen/settings/AdvancedSettingsScreen.kt

```diff
@@ -138,16 +137,39 @@ fun AdvancedSettingsScreen(
             )
 
             GroupHeader(stringResource(R.string.debugging))
+            val exportDebugLogsLauncher =
+                rememberLauncherForActivityResult(ActivityResultContracts.CreateDocument("text/plain")) {
+                    it?.let(vm::exportDebugLogs)
+                }
             SettingsListItem(
-                headlineContent = stringResource(R.string.about_device),
-                supportingContent = """
-                    **Version**: ${BuildConfig.VERSION_NAME} (${BuildConfig.VERSION_CODE})
-                    **Build type**: ${BuildConfig.BUILD_TYPE}
-                    **Model**: ${Build.MODEL}
-                    **Android version**: ${Build.VERSION.RELEASE} (${Build.VERSION.SDK_INT})
-                    **Supported Archs**: ${Build.SUPPORTED_ABIS.joinToString(", ")}
-                    **Memory limit**: $memoryLimit
+                headlineContent = stringResource(R.string.debug_logs_export),
+                modifier = Modifier.clickable { exportDebugLogsLauncher.launch(vm.debugLogFileName) }
+            )
+            val clipboard = context.getSystemService(CLIPBOARD_SERVICE) as ClipboardManager
+            val deviceContent = """
+                    Version: ${BuildConfig.VERSION_NAME} (${BuildConfig.VERSION_CODE})
+                    Build type: ${BuildConfig.BUILD_TYPE}
+                    Model: ${Build.MODEL}
+                    Android version: ${Build.VERSION.RELEASE} (${Build.VERSION.SDK_INT})
+                    Supported Archs: ${Build.SUPPORTED_ABIS.joinToString(", ")}
+                    Memory limit: $memoryLimit
                 """.trimIndent()
+            SettingsListItem(
+                modifier = Modifier.combinedClickable(
+                    onLongClickLabel = stringResource(R.string.copy_to_clipboard),
+                    onLongClick = {
+                        haptics.performHapticFeedback(HapticFeedbackType.LongPress)
+                        clipboard.setPrimaryClip(
+                            ClipData.newPlainText("Device Information", deviceContent)
+                        )
+
+                        Toast.makeText(context, R.string.toast_copied_to_clipboard, Toast.LENGTH_SHORT).show()
+                    }
+                ) {
+
+                },
```

Comment:
I think doing `onClick = {}` would be more readable

Reply:
Done